apps/flingo/src/styles/mat-slide-toggle.scss

```diff
@@ -0,0 +1,11 @@
+.mat-slide-toggle.mat-checked {
```

Comment:
Didn't quite know where to add this global overwrite so I made a new file for it? Should we rather have a `material.scss` file for all global angular material overwrites? @ubergeoff